packages/lesswrong/components/posts/PostsPage/PostsPage.tsx

```diff
@@ -443,11 +443,9 @@ const PostsPage = ({post, eagerPostComments, refetch, classes}: {
     limit: 1000
   });
   
-  useOnNotificationsChanged(currentUser, (message) => {
-    if (message.eventType === 'notificationCheck') {
-      if (currentUser && isDialogueParticipant(currentUser._id, post)) {
-        refetchDebateResponses();
-      }
+  useOnServerSentEvent('notificationCheck', currentUser, (message) => {
+    if (currentUser && isDialogueParticipant(currentUser._id, post)) {
```

Comment:
This is a nit as it's pre-existing before this PR, but `isDialogueParticipant` really shouldn't live in this file (probably `lib/posts/helpers.ts` is better).